examples/next/pages/ui/components/account-settings/change-password/index.page.tsx

```diff
@@ -10,38 +10,40 @@ import {
   Flex,
   Heading,
   AccountSettings,
+  withAuthenticator,
 } from '@aws-amplify/ui-react';
 import '@aws-amplify/ui-react/styles.css';
 
 import awsExports from './aws-exports';
 Amplify.configure(awsExports);
 
-export default function App() {
+function App({ signOut }) {
   const [isSuccessful, setIsSuccessful] = React.useState(false);
+
   return (
     <Authenticator>
```

Comment:
wait, isn't the Authenticator nested inside the Authenticator now?

Reply:
Oh gosh, thank you for the catch. This actually worked because the double authenticator will just render its children.

Reply:
Fixed in 6210b57